build_tools/cmake/iree_native_test.cmake

```diff
@@ -26,8 +26,10 @@ include(CMakeParseArguments)
 # ARGS: additional arguments passed to the test binary. TEST_INPUT_FILE_ARG and
 #     --device=DRIVER are automatically added if specified.
 # SRC: binary target to run as the test.
+# XFAIL: The target will run, but its pass/fail status will be inverted.
```

Comment:
Nit: I think it's probably better to use CMake's `WILL_FAIL` terminology here since this has the exact same effect as the arg to `add_test`

Reply:
Done.